fabfile.py

```diff
@@ -50,12 +53,22 @@ def migrate(name_config=None, debug=False):
     """
     if debug:
         utils.configure_logging("DEBUG")
-    load_config(name_config)
+    try:
+        load_config(name_config)
+    except oslo.config.cfg.Error:
+        traceback.print_exc()
+        sys.exit(ERROR_INVALID_CONFIGURATION)
     env.key_filename = cfglib.CONF.migrate.key_filename
     env.connection_attempts = cfglib.CONF.migrate.ssh_connection_attempts
-    cloud = cloud_ferry.CloudFerry(cfglib.CONF)
-    cloud.migrate(Scenario(path_scenario=cfglib.CONF.migrate.scenario,
-                           path_tasks=cfglib.CONF.migrate.tasks_mapping))
+    try:
+        cloud = cloud_ferry.CloudFerry(cfglib.CONF)
+    except oslo.config.cfg.Error:
```

Comment:
Can we move `load_config(name_config)` inside this `try ... except` block to avoid duplicate logic?